New user man: First version (#940)
Avaiga/taipy-doc: docs/manuals/userman/what-if-analysis/index.md

In this section, we explore how to use *scenarios* for what-if analysis in Taipy.
What-if analysis is all about exploring different scenarios to see how variable
changes affect outcomes. It's essentially asking "What if?" questions and examining
the results.

# What is a Scenario?
A `Scenario^` in Taipy represents an instance of a business problem. Among other features

> scenarios own data nodes. Two different scenarios can own two alternative data nodes

Reviewer: The sentence doesn't make sense to me. Is this what you mean?
```suggestion
A `Scenario^` in Taipy represents an instance of a business problem.
Scenarios own data nodes, and two different scenarios can own two alternative data nodes
```

(from the same `DataNodeConfig^`). This is particularly handy when you want to represent

> different situation, assumption or condition, and compare the data between them.

Reviewer: ```suggestion
(from the same `DataNodeConfig^`). This is particularly handy when you want to represent
different situations, assumptions or conditions, and compare the data between them.
```

    or parameter set. It is essential for use cases where experiments need to be
    conducted with different assumptions, or when data needs to be preserved for
    auditing and comparison purposes.<br>
    For more details, see the [multiple scenarios](multiple-scenarios.md) page.

- **Temporal Analysis**: Use cycles and data node scope for recurring business
    cases, ideal for time-based analyses like weekly forecasts, monthly planning,
    etc.<br>
    For more details, see the [recurrent scenarios](scenarios-and-cycles.md) page.

- **Easy to use**: Easily manage and compare multiple scenarios for various
    business situations.<br>
    For more details, see the [scenario comparison](scenario-comparison.md) page.

# How to use Scenarios?

What-if analysis in Taipy involves configuring *data nodes*, setting up *scenario* configurations,
and instantiating new *scenarios* at runtime. Each *scenario* holding an alternative data node,
represents a different situation, assumption or condition, and as an end-user you can compare
the different scenarios to understand the impact of various decisions, or events on your data.

> Here is the main principle:

Reviewer: ```suggestion
Here are the main principles:
```
or
```suggestion
Here are the main steps:
```

> 1. **Configuring data nodes**: The first step consist in configuring the data nodes that will be

Reviewer: ```suggestion
1. **Configuring data nodes**: The first step consists in configuring the data nodes that will be
```
